instrumentation/spring/spring-batch-3.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/spring/batch/v3_0/jsr/CustomEventChunkListener.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.spring.batch.v3_0.jsr;
+
+import io.opentelemetry.api.trace.Span;
+import javax.batch.api.chunk.listener.ChunkListener;
+
+public class CustomEventChunkListener implements ChunkListener {
```

Comment:
```suggestion
class CustomEventChunkListener implements ChunkListener {
```

Reply:
theres still a lot of public classes, do they need to be public or could they be package private?

Reply:
yes, the remaining public classes are loaded via reflection